lib/inactive_users_suspension_reminder.rb

```diff
@@ -25,8 +26,12 @@ def send_reminders
         Rails.logger.warn "#{self.class}: Failed to send suspension reminder email to #{user.email}."
         notify_airbrake(e, user)
       rescue => e
-        Rails.logger.warn "#{self.class}: #{e.response.error.message} while sending email to #{user.email}."
         notify_airbrake(e, user)
+        begin
+          Rails.logger.warn "#{self.class}: #{e.response.error.message} while sending email to #{user.email}."
+        rescue NoMethodError
```

Comment:
Instead of waiting for an error to be raised, maybe we could do something like:
```ruby
if e.responds_to?(:response)
  Rails.logger.warn "#{self.class}: #{e.response.error.message} while sending email to #{user.email}."
else
  Rails.logger.warn "#{self.class}: #{e.message} while sending email to #{user.email}."
end
```